ftw/simplelayout/tests/test_simplelayout_portlet.py

```diff
@@ -0,0 +1,93 @@
+from ftw.builder import Builder
+from ftw.builder import create
+from ftw.simplelayout.interfaces import IPageConfiguration
+from ftw.simplelayout.portlets.portlet import Assignment
+from ftw.simplelayout.testing import FTW_SIMPLELAYOUT_FUNCTIONAL_TESTING
+from ftw.simplelayout.testing import SimplelayoutTestCase
+from ftw.testbrowser import browsing
+from plone.app.textfield.value import RichTextValue
+from plone.portlets.interfaces import IPortletAssignmentMapping
+from plone.portlets.interfaces import IPortletManager
+from plone.uuid.interfaces import IUUID
+from zope.component import getMultiAdapter
+from zope.component import getUtility
+import transaction
+
+
+class TestSimplelayoutView(SimplelayoutTestCase):
+
+    layer = FTW_SIMPLELAYOUT_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        self.setup_sample_ftis(self.portal)
+        self.setup_block_views()
+
+        self.container = create(Builder('sample container'))
```

Comment:
why are there sample types btw? couldn't we just use the normal page and text block?